doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,468 @@
+
+    >>> from scipy import misc
+    >>> img = misc.face()
+
+If you would rather use your own image, in order to transform this image into a
+numpy array that can be manipulated, we will use the ``imread`` function from the
+`matplotlib.image` submodule.
+
+Now, ``img`` is a numpy array, as we can see when using the ``type`` function:
+
+::
+
+    >>> type(img)
+    numpy.ndarray
+
+We can see the image using the `matplotlib.pyplot.imshow` function:
+
+    >>> import matplotlib.pyplot as plt
+    >>> plt.imshow(img)
+
+.. image:: https://docs.scipy.org/doc/scipy/reference/_images/scipy-misc-face-1.png
+
+Shape, axis and array properties
+--------------------------------
+
+One particular important observation to be made is that, if you are at all
+familiar with linear algebra, you might expect `dimensions` to refer to the
+number of coordinates in each axis of your coordinate system; thus, a
+two-dimensional vector would be a list of two coordinates. In NumPy, however,
+a `dimension` is one of the axes in the array, so a two-dimensional vector
+is the equivalent of a matrix.
+
+First, let's check for the shape of the data in our array. Since this image is
+a two-dimensional set of data (the pixels in the image form a rectangle), we
+might expect a two-dimensional array to represent it (a matrix). However, using
+the ``shape`` property of this NumPy array gives us a different result:
+
+::
+
+    >>> img.shape
+    (768, 1024, 3)
+
+The output is a tuple, with three elements, which means that this is a
+three-dimensional array. In fact, since this is a color image, we have one set
+of pixel data per color (in this case, red, green and blue - RGB). You can see
+this by looking at the shape above: it indicates that we have 3 sets of 768x1024
+matrices.
+
+Furthermore, using the ``ndim`` property of this array, we can see that
+
+::
+
+    >>> img.ndim
+    3
+
+Each dimension is also referred to as an `axis` on NumPy. Because of how \
+color images work, the *first index in the 3rd axis* is the red pixel data for
+our image. We can access this by using the syntax
+
+::
+
+    >>> img[:, :, 0]
+    array([[121, 138, 153, ..., 119, 131, 139],
+           [ 89, 110, 130, ..., 118, 134, 146],
+           [ 73,  94, 115, ..., 117, 133, 144],
+           ...,
+           [ 87,  94, 107, ..., 120, 119, 119],
+           [ 85,  95, 112, ..., 121, 120, 120],
+           [ 85,  97, 111, ..., 120, 119, 118]], dtype=uint8)
+
+From the output above, we can see that every value in ``img[:,:,0]`` is an
+integer value between 0 and 255, representing the level of red in each
+corresponding image pixel.
+
+As expected, this is a 768x1024 matrix:
+
+::
+
+    >>> img[:, :, 0].shape
+    (768, 1024)
+
+Since we are going to perform linear algebra on this data, it might be more
+interesting to have real numbers between 0 and 1 in each entry of the matrices
+to represent the RGB values. We can do that by setting
+
+    >>> img_array = img/255
```

Comment:
This works, but relies on knowing about the input scale. Perhaps it's worth pointing out that one should use, e.g., https://scikit-image.org/docs/dev/api/skimage.util.html#skimage.util.img_as_float. Not sure though if it complicates things too much here. @stefanv?

Reply:
I have reformulated this a bit to try and make it a little more explicit that it's a special case. The idea here is to actually show that you can do that kind of operation - divide an entire array by a scalar - and broadcasting will make it work. I'm aware that it's not ideal in the real world. Let me know what you think

Reply:
this is nicer, but would still be useful I think to add a small note somewhere about scikit-image having a utility function for this. then we can call it good imho

Reply:
Let me know if this is better!